src/gsy_e/gsy_e_core/util.py

```diff
@@ -158,8 +159,8 @@ def get_char(self, timeout=0):
         return False
 
 
-def make_iaa_name(owner):
-    return f"IAA {owner.name}"
+def make_ma_name(owner):
+    return f"MA {owner.name}"
```

Comment:
This method is the equivalent of this on GSy-framework: https://github.com/gridsingularity/gsy-framework/pull/299/files/728943715ee3cddca74c7eaa0204dca63aa2bf85#diff-e0a72d7786b41a5338bfdd2b84dd8ecc663518e42e3c82e2038dc65af7c4ab91R363-R365
The difference is that here we deal with objects and not dictionaries. IMO this could be unified (so that we would have only one source of truth). 
If you want, I can work on it. I would keep it for a separate PR though.

Reply:
IMO not important and can be dealt with in the future,